common/app/utils/No-Props-Passthrough.jsx

```diff
@@ -0,0 +1,12 @@
+import { PropTypes } from 'react';
+
+// use when passing a react primitive element as a child to a
+// react-boostrap component that will inject props
+// using cloneElement
+export default function NoPropsPassthrough({ children }) {
+  return children;
+}
+
+NoPropsPassthrough.propTypes = {
+  chilren: PropTypes.element
```

Comment:
You teach me how to code by picking up on everything, I will gloat over a typo! :smile: